spec/serializers/api/v2/green_lanes/category_assessment_serializer_spec.rb

```diff
@@ -37,6 +40,12 @@
               { id: '1', type: 'additional_code' },
             ],
           },
+          "geographical_area": {
+            "data": {
+              "id": 'EU',
+              "type": 'geographical_area',
+            },
+          },
```

Comment:
This looks like it should include the following as well - not sure if this is just the test still passing but its there, or if the content is missing?

```json
"excluded_geographical_areas": {
  "data": [
  
  ],
},
```